themes/structure/static/js/script.js

```diff
@@ -1,39 +0,0 @@
-const body = document.querySelector("body");
```

Comment:
We should bring this back and just import it correctly. Did you try that? Why didn't it work?

Reply:
I did that before and it was working. But when I added more pages and added more js (for those pages) to it, they started to interfere with each other i.e the js written for one file will error out on another file and cause that one's js to not work. I don't know if you get what I mean?
So I just removed the general script.js file and put the js codes in script tags in their respective HTML files.

Reply:
We've established above that we want to keep the js files in public/js. What you can then do is create a js file for every special function that's being re-used in multiple files, to prevent the interference, while _also_ stopping the repetition.